packages/nestjs-access-control/src/access-control.module-definition.ts

```diff
@@ -76,6 +76,7 @@ export function createAccessControlProviders(overrides: {
 }): Provider[] {
   return [
     ...(overrides.providers ?? []),
+    AccessControlService,
```

Comment:
this is already provided by the `createAccessControlServiceProvider()` method. please review all of the changes where you possibly repeated this pattern.